tensorflow/python/ops/parallel_for/control_flow_ops_test.py

```diff
@@ -97,6 +97,16 @@ def loop_fn(i):
       self._test_loop_fn(loop_fn, 3, fallback_to_while_loop=False)
     self._test_loop_fn(loop_fn, 3, fallback_to_while_loop=True)
 
+  def test_defun(self):        
+    def_function.function
```

Comment:
Forgot the `@` (for the decorator)

Reply:
Yes it was a typo but now I've removed the function here and keep it only in the parent function as  I think probably it is a better test.

Reply:
Now it is totally failing (no session etc..). Do we need to limit this test to graph mode only?